crates/e2e/tests/e2e/quote_verification.rs

```diff
@@ -199,7 +198,7 @@ async fn test_bypass_verification_for_rfq_quotes(web3: Web3) {
     let verified_quote = Estimate {
         out_amount: U256::from(16380122291179526144u128),
         gas: 225000,
-        solver: H160::from_str("0xe3067c7c27c1038de4e8ad95a83b927d23dfbd99").unwrap(),
+        solver: address!("e3067c7c27c1038de4e8ad95a83b927d23dfbd99").into_legacy(),
```

Comment:
A bit odd to switch to the new macro and then convert it back to the old type. 😅

Reply:
I prefer it like this because eventually we can kill the trait and know where it's failing

Reply:
Inversely we will migrate the `solver` type to `alloy::primitives::Address` and see where it's failing. 😅 
I don't feel strongly about this - just thought it was a bit odd. Feel free to ignore this nit.